juju/juju-gui-charm
# Add new jujugui dependencies to deps directory.

QA:

```
juju bootstrap ec2
make deploy
juju ssh juju-gui/0
python
  >> import jujugui
```

Should not blow up.
